template/src/ontology/Makefile.jinja2

```diff
@@ -566,6 +570,10 @@ mirror-{{ ont.id }}: | $(TMPDIR)
 		$(ROBOT) convert -i $(MIRRORDIR)/{{ ont.id }}.owl.gz -o $@.tmp.owl{%- if ont.make_base %} && \
 		$(ROBOT) remove -i $@.tmp.owl {% if ont.base_iris is not none %}{% for iri in ont.base_iris %}--base-iri {{iri}} {% endfor %}{% else %}--base-iri $(URIBASE)/{{ ont.id.upper() }} {% endif %}--axioms external --preserve-structure false --trim false -o $@.tmp.owl{% endif %} &&\
 		mv $@.tmp.owl $(TMPDIR)/$@.owl; fi
+{%- elif 'custom' == ont.module_type and 'custom' == ont.mirror_type %}
+mirror-{{ ont.id }}: 
+	echo "ERROR: You have configured your default mirror type to be custom; this behavior needs to be overwritten in {{ project.id }}.Makefile!" && false
+{%- elif 'custom' == ont.module_type and 'none' == ont.mirror_type -%}
```

Comment:
Why do you need to couple the `module_type` and the `mirror_type` here so tightly? Couldnt the be configured separately?

Reply:
I removed the check for both parameters.